lms/djangoapps/grades/models.py

```diff
@@ -37,14 +37,31 @@
 BlockRecord = namedtuple('BlockRecord', ['locator', 'weight', 'raw_possible', 'graded'])
 
 
-class BlockRecordList(tuple):
+class BaseTuple(tuple):
     """
-    An immutable ordered list of BlockRecord objects.
+    Wrapper class for tuple to allow overrides for both __new__ and __init__
+    
+    Normally overriding both methods will produce a deprecation warning.  This will
+    resolve that error and maintain existing behavior.
+        See https://bugs.python.org/issue1683368
```

Comment:
Is this no longer needed when we upgrade to Python 3?  If so, can we add a comment to remind us to remove this then?